### DISCUSSION TITLE
Cannot open DNG file.

### DISCUSSION THREAD

**Discussion by klydeid** [OpenImageIO/oiio#3631] [Q&A]
is this raw file not supported?
i have tested another Raw format file. it's all ok except DNG file.

my OIIO version is 2.3.15.
libraw: 0.20.2
DNG sdk: 1.4

here is sample DNG file:   
https://www.dropbox.com/s/810gz4fv2u8mxko/_MG_9254.dng?dl=0

**Comment by lgritz** [comment]
I downloaded that file and tried the following commands:

```
$ oiiotool -info -v IMG_9254.dng
Reading IMG_9254.dng
IMG_9254.dng         : 4368 x 2912, 3 channel, uint16 raw
    channel list: R, G, B
    DateTime: "2012-08-15 15:11:56"
    ExposureTime: 0
    FNumber: 0
    Make: "TIFF"
    Orientation: 1 (normal)
    PixelAspectRatio: 1
    Software: "Adobe Photoshop Lightroom 4.1 (Windows)"
    Exif:ApertureValue: -inf (f/0.0)
    Exif:ExifVersion: ""
    Exif:Flash: 0 (no flash)
    Exif:FocalLength: 0 (0 mm)
    Exif:ISOSpeedRatings: 0
    Exif:ShutterSpeedValue: inf (1/inf s)
    oiio:ColorSpace: "sRGB"
    raw:cam_mul: 0, 1, 0, 0
    raw:Demosaic: "AHD"
    raw:HighlightMode: 0
    raw:pre_mul: 2.78921, 2.78925, 2.78933, 0
    raw:rgb_cam: 2.10548, -0.662467, -0.443018, 0, -0.221916, 1.20641, 0.0155049, 0, -0.0150022, -0.137836, 1.15284, 0
    TIFF:AFPoint: -1
    TIFF:FocalUnits: 1
    TIFF:ImageStabilization: -1
    TIFF:LensID: 18446744073709551615
```

and I also tried

    `iv IMG_9254.dng`

and it displayed reasonably for me.

So here's what I'd suggest next:

    oiiotool --help

This will produce a long output, but at some point near the end you will see "Input formats supported:" and also "Dependent libraries:". Can you verify that "raw" is in the list of formats, and also that the dependent libraries include libraw (and tell us which version it thinks it got)?
